plugins/node/opentelemetry-instrumentation-aws-sdk/test/dynamodb.test.ts

```diff
@@ -1,1136 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-import {
-  AwsInstrumentation,
-  CommandInput,
-  AwsSdkDynamoDBStatementSerializer,
-} from '../src';
-import {
-  getTestSpans,
-  registerInstrumentationTesting,
-} from '@opentelemetry/contrib-test-utils';
-const instrumentation = registerInstrumentationTesting(
-  new AwsInstrumentation()
-);
-import * as AWS from 'aws-sdk';
-import { AWSError } from 'aws-sdk';
-
-import { mockV2AwsSend } from './testing-utils';
-import {
-  DBSYSTEMVALUES_DYNAMODB,
-  SEMATTRS_AWS_DYNAMODB_ATTRIBUTE_DEFINITIONS,
-  SEMATTRS_AWS_DYNAMODB_CONSISTENT_READ,
-  SEMATTRS_AWS_DYNAMODB_CONSUMED_CAPACITY,
-  SEMATTRS_AWS_DYNAMODB_COUNT,
-  SEMATTRS_AWS_DYNAMODB_EXCLUSIVE_START_TABLE,
-  SEMATTRS_AWS_DYNAMODB_GLOBAL_SECONDARY_INDEX_UPDATES,
-  SEMATTRS_AWS_DYNAMODB_GLOBAL_SECONDARY_INDEXES,
-  SEMATTRS_AWS_DYNAMODB_INDEX_NAME,
-  SEMATTRS_AWS_DYNAMODB_ITEM_COLLECTION_METRICS,
-  SEMATTRS_AWS_DYNAMODB_LIMIT,
-  SEMATTRS_AWS_DYNAMODB_LOCAL_SECONDARY_INDEXES,
-  SEMATTRS_AWS_DYNAMODB_PROJECTION,
-  SEMATTRS_AWS_DYNAMODB_PROVISIONED_READ_CAPACITY,
-  SEMATTRS_AWS_DYNAMODB_PROVISIONED_WRITE_CAPACITY,
-  SEMATTRS_AWS_DYNAMODB_SCAN_FORWARD,
-  SEMATTRS_AWS_DYNAMODB_SCANNED_COUNT,
-  SEMATTRS_AWS_DYNAMODB_SEGMENT,
-  SEMATTRS_AWS_DYNAMODB_SELECT,
-  SEMATTRS_AWS_DYNAMODB_TABLE_COUNT,
-  SEMATTRS_AWS_DYNAMODB_TABLE_NAMES,
-  SEMATTRS_AWS_DYNAMODB_TOTAL_SEGMENTS,
-  SEMATTRS_DB_NAME,
-  SEMATTRS_DB_OPERATION,
-  SEMATTRS_DB_STATEMENT,
-  SEMATTRS_DB_SYSTEM,
-} from '@opentelemetry/semantic-conventions';
-import { expect } from 'expect';
-import type { ConsumedCapacity as ConsumedCapacityV2 } from 'aws-sdk/clients/dynamodb';
-import type { ConsumedCapacity as ConsumedCapacityV3 } from '@aws-sdk/client-dynamodb';
-import * as sinon from 'sinon';
-
-type ConsumedCapacity = ConsumedCapacityV2 | ConsumedCapacityV3;
-
-const responseMockSuccess = {
-  requestId: '0000000000000',
-  error: null,
-};
-
-describe('DynamoDB', () => {
```

Comment:
there's now almost no coverage anymore on `plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/dynamodb.ts` - was this intentional?

Reply:
Yes. The existing tests in `dynamodb.test.ts` are specific to JS SDK v2 whose support is being dropped.
There are no tests for DynamoDB in JS SDK v3, but that's outside the scope of this PR.